.env.example

```diff
@@ -56,3 +56,10 @@ XAI_API_KEY=
 
 # Include this environment variable if you want more logging for debugging locally
 VITE_LOG_LEVEL=debug
+
+# Example Context Values for qwen2.5-coder:32b
+# 
+# DEFAULT_NUM_CTX=32768 # Consumes 36GB of VRAM
```

Comment:
Also, this will need to live anywhere else that the env vars are being utilized, Dockerfile and docker-compose files for example. Other than that this looks good to me.